File: Examenes/22-1/JAVA/EduSoftLP2Model/src/pe/edu/pucp/edusoftlp2/model/Docente.java
package pe.edu.pucp.edusoftlp2.model;
public class Docente {
    private int idDocente;
    private String codigoPUCP;
    private String nombre;
    private String apellidoPaterno;
    private TipoDocumento tipoDocumento;
    private String numeroDocumento;
    private String maximoGradoAlcanzado;

    public int getIdDocente() {
        return idDocente;
    }

    public void setIdDocente(int idDocente) {
        this.idDocente = idDocente;
    }

    public String getCodigoPUCP() {
        return codigoPUCP;
    }

    public void setCodigoPUCP(String codigoPUCP) {
        this.codigoPUCP = codigoPUCP;
    }

    public String getNombre() {
        return nombre;
    }

    public void setNombre(String nombre) {
        this.nombre = nombre;
    }

    public String getApellidoPaterno() {
        return apellidoPaterno;
    }

    public void setApellidoPaterno(String apellidoPaterno) {
        this.apellidoPaterno = apellidoPaterno;
    }

    public TipoDocumento getTipoDocumento() {
        return tipoDocumento;
    }

    public void setTipoDocumento(TipoDocumento tipoDocumento) {
        this.tipoDocumento = tipoDocumento;
    }

    public String getNumeroDocumento() {
        return numeroDocumento;
    }

    public void setNumeroDocumento(String numeroDocumento) {
        this.numeroDocumento = numeroDocumento;
    }

    public String getMaximoGradoAlcanzado() {
        return maximoGradoAlcanzado;
    }

    public void setMaximoGradoAlcanzado(String maximoGradoAlcanzado) {
        this.maximoGradoAlcanzado = maximoGradoAlcanzado;
    }
    
    
}

File: Examenes/EX1/22-1/CSharp/EduSoftLP2/EduSoftLP2Model/Docente.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace EduSoftLP2Model
{
    public class Docente
    {
        private int _idDocente;
        private string _codigoPUCP;
        private string _nombre;
        private string _apellidoPaterno;
        private TipoDocumento _tipoDocumento;
        private string _numeroDocumento;
        private string _maximoGradoAlcanzado;
        public int IdDocente { get => _idDocente; set => _idDocente = value; }
        public string CodigoPUCP { get => _codigoPUCP; set => _codigoPUCP = value; }
        public string Nombre { get => _nombre; set => _nombre = value; }
        public string ApellidoPaterno { get => _apellidoPaterno; set => _apellidoPaterno = value; }
        public TipoDocumento TipoDocumento { get => _tipoDocumento; set => _tipoDocumento = value; }
        public string NumeroDocumento { get => _numeroDocumento; set => _numeroDocumento = value; }
        public string MaximoGradoAlcanzado { get => _maximoGradoAlcanzado; set => _maximoGradoAlcanzado = value; }
    }
}
